crates/astria-sequencer/src/service/mempool.rs

```diff
@@ -74,7 +74,18 @@ fn handle_check_tx(req: request::CheckTx) -> response::CheckTx {
             };
         }
     };
-    let signed_tx = SignedTransaction::try_from_raw(raw_signed_tx).unwrap();
```

Comment:
This is really orthogonal to the rest...

Reply:
should i add to a different PR? sorry i was just running some tests and also found this :p

Reply:
Yeah, that would be much cleaner. 🙏

Reply:
https://github.com/astriaorg/astria/pull/418